lib/core/styles/_variables.scss

```diff
@@ -6,3 +6,6 @@ $animation-curve-default: $animation-curve-fast-out-slow-in !default;
 
 $swift-ease-in-duration: 300ms !default;
 $swift-ease-in-timing-function: cubic-bezier(0.55, 0, 0.55, 0.2) !default;
+
+$datatable-background: #ffffff;
```

Comment:
you should be able to fetch the color from the material palette
 background: map_get($mat-grey, A100)
https://github.com/angular/material2/blob/master/src/lib/core/theming/_palette.scss#L611

Instead f5f5f5 should be 
 background: map_get($mat-grey, 100)